types/xrm/index.d.ts

```diff
@@ -6159,11 +6159,11 @@ declare namespace Xrm {
     /**
      * Interface for the WebAPI RetrieveMultiple request response
      */
-    interface RetrieveMultipleResult {
+    interface RetrieveMultipleResult<T = any> {
```

Comment:
Hmm... not that I expect it to be that big of an issue, but this is a breaking change isn't it?

Reply:
It shouldn't be. The Generic parameter has a default of `any` specified. And the [Typescript Documentation](https://www.typescriptlang.org/docs/handbook/2/generics.html#generic-parameter-defaults) specifies that if you don't specify the parameter it will just default to `any`.

So typing something as `Xrm.RetrieveMultipleResult` would be the exact same as `Xrm.RetrieveMultipleResult<any>`. And since both are considered the same it shouldn't be a Breaking Change. But now we also gain the benefits of allowing a person to override that parameter to specify exactly what type that Dataverse is returning, which should allow for developers to make less error-prone code.